DOCS-2509: Document create separate tflite module (#3655)
viamrobotics/docs: docs/appendix/changelog.md

<!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->

{{% changelog date="2024-11-05" color="added" title="TFLite model moved to module" %}}

> The TFLite CPU ML model service is now supported as a [module](https://app.viam.com/module/viam/tflite_cpu).

Author: ```suggestion
The `TFLite CPU` ML model service is now supported as a [module](https://app.viam.com/module/viam/tflite_cpu).
```

> The `TFLite CPU` ML model service is now supported as a [module](https://app.viam.com/module/viam/tflite_cpu).

Reviewer: It's not really supported as a module so much as that it is in a module now
```suggestion
The `TFLite CPU` ML model service has moved to a [module](https://app.viam.com/module/viam/tflite_cpu).
```
